packages/mobile/src/account/Settings.tsx

```diff
@@ -0,0 +1,344 @@
+}
+
+interface StateProps {
+  account: string | null
+  e164PhoneNumber: string | null
+  devModeActive: boolean
+  analyticsEnabled: boolean
+  numberVerified: boolean
+  pincodeType: PincodeType
+  backupCompleted: boolean
+  lockWithPinEnabled: boolean
+  fornoEnabled: boolean
+  gethStartedThisSession: boolean
+  preferredCurrencyCode: LocalCurrencyCode
+}
+
+type OwnProps = StackScreenProps<StackParamList, Screens.Settings>
+
+type Props = StateProps & DispatchProps & WithTranslation & OwnProps
+
+const mapStateToProps = (state: RootState): StateProps => {
+  return {
+    backupCompleted: state.account.backupCompleted,
+    account: state.web3.account,
+    devModeActive: state.account.devModeActive || false,
+    e164PhoneNumber: state.account.e164PhoneNumber,
+    analyticsEnabled: state.app.analyticsEnabled,
+    numberVerified: state.app.numberVerified,
+    pincodeType: pincodeTypeSelector(state),
+    lockWithPinEnabled: state.app.lockWithPinEnabled,
+    fornoEnabled: state.web3.fornoMode,
+    gethStartedThisSession: state.geth.gethStartedThisSession,
+    preferredCurrencyCode: getLocalCurrencyCode(state),
+  }
+}
+
+const mapDispatchToProps = {
+  revokeVerification,
+  setNumberVerified,
+  resetAppOpenedState,
+  setAnalyticsEnabled,
+  toggleBackupState,
+  devModeTriggerClicked,
+  setLockWithPinEnabled,
+  toggleFornoMode,
+}
+
+interface State {
+  fornoSwitchOffWarning: boolean
+  promptFornoModal: boolean
+}
+
+export class Account extends React.Component<Props, State> {
+  componentDidMount() {
+    const promptFornoModal = this.props.route.params?.promptFornoModal
+    if (promptFornoModal) {
+      this.setState({
+        promptFornoModal,
+      })
+    }
+  }
+
+  goToProfile = () => {
+    CeloAnalytics.track(CustomEventNames.edit_profile)
+    this.props.navigation.navigate(Screens.Profile)
+  }
+
+  goToLanguageSetting = () => {
+    this.props.navigation.navigate(Screens.Language, { fromSettings: true })
+  }
+
+  goToLocalCurrencySetting = () => {
+    this.props.navigation.navigate(Screens.SelectLocalCurrency)
+  }
+
+  goToLicenses = () => {
+    this.props.navigation.navigate(Screens.Licenses)
+  }
+
+  goToSupport = () => {
+    this.props.navigation.navigate(Screens.Support)
+  }
+
+  resetAppOpenedState = () => {
+    this.props.resetAppOpenedState()
+    Logger.showMessage('App onboarding state reset.')
+  }
+
+  toggleNumberVerified = () => {
+    this.props.setNumberVerified(!this.props.numberVerified)
+  }
+
+  revokeNumberVerification = async () => {
+    if (this.props.e164PhoneNumber && !isE164Number(this.props.e164PhoneNumber)) {
+      Logger.showMessage('Cannot revoke verificaton: number invalid')
+      return
+    }
+    Logger.showMessage(`Revoking verification`)
+    this.props.revokeVerification()
+  }
+
+  toggleBackupState = () => {
+    this.props.toggleBackupState()
+  }
+
+  showDebugScreen = () => {
+    this.props.navigation.navigate(Screens.Debug)
+  }
+
+  getDevSettingsComp() {
```

Comment:
probably worth breaking this out into a separate component, but naming it `renderDevSettings` would be a bit nicer (also i like this article https://dev.to/jcutrell/react-anti-pattern-renderthing-50nd)

Reply:
It make sense for me, but I would do it in a separate PR, not as a part of Redesign.